logview/backend/routes/api/userRoutes.js

```diff
@@ -43,6 +43,16 @@ router.post(
 	async (req, res, next) => {
 		try {
 			const data = await userService.create(req.body);
+			const msg = {
+				subject: 'Confirm email address. Observer BSA 2018',
+				html: `<p>You are receiving this because you (or someone else) have requested the registration new account.</p>
+		               <p>Please click on the following link, or paste this into your browser to complete the activation:</p>
+		               <p>http://${req.headers.host}/confirm/${
+					data.userActivationToken
```

Comment:
Should be like in the change PW email:
<p>http://${req.headers.host}/confirm/?activationToken=${data.userActivationToken}</p>